3_wtemp/src/count_wtemp_sites.R

```diff
@@ -0,0 +1,26 @@
+count_wtemp_sites <- function(nwis_inv_ind, wqp_inv_ind) {
+  # Use the NWIS inventory to learn about continuous water temperature data.
+  # Eventually it'd be nice to pull down all the NWIS samples, but until then,
+  # we need to take shortcuts to approximately determine which sites have good
+  # continuous monitoring coverage
+  nwis_inventory <- readRDS(sc_retrieve(nwis_inv_ind))
+  continuous <- nwis_inventory %>%
+    filter(end_date - begin_date >= as.difftime(3000, units='days')) %>% # arbitrary choice to define "continuous" as sites having monitoring occurring over a 3000-day range
```

Comment:
should we pull this out into a cfg somewhere?

Reply:
the definition of continuous, that is

Reply:
Yes, eventually...but I'd rather wait until we've settled into a decision about the structure of this decision. Right now the choice is between 3000 and some other number, but we could provide a much better answer if we pulled enough NWIS data to make the choice be something more like number of days with ~>24 samples, number of ~complete days per year, number of ~complete years in the record, etc.. So I'm going to keep that idea on hold for now.